docs/src/content/docs/basics/terminology.mdx

```diff
@@ -12,7 +12,11 @@ A service that retrieves data from the user's *virtual agent*, anonymizes it, an
 
 ## Agent SDK
 
-Platform-independent core logic integrated by an *app* that enables them to be compatible with the *Mee data network.*  
+A software development kit that (i) provides the core logic of an *identity agent* and (ii) enables a *local* *app* to become interoperable with the *Mee data network*. The *agent SDK* manages *personal information* that may be (i) created from the user's data entry into their *identity agent* (ii) created by an app (or co-created by an *app* during interactions with the user) or (iii) received over the *Mee data network* from another *identity* *agent*.
```

Comment:
Noticed the duplication of the Agent SDK sections and replaced with the one which I liked most

Reply:
I agree with this change